spec/features/consumer/registration_spec.rb

```diff
@@ -6,7 +6,20 @@
 
   describe "Registering a Profile" do
     let(:user) { create(:user, password: "password", password_confirmation: "password") }
-    before { Spree::Config.enterprises_require_tos = false }
+
+    before do
+      Spree::Config.enterprises_require_tos = false
+
+      albania = Spree::Country.create!({ name: "Albania", iso3: "ALB", iso: "AL", iso_name: "ALBANIA", numcode: "8" }, without_protection: true)
+      Spree::State.create!({ name: "Berat", abbr: "BRA", country: albania }, without_protection: true)
+      Spree::Country.create!({ name: "Chad", iso3: "TCD", iso: "TD", iso_name: "CHAD", numcode: "148" }, without_protection: true)
+    end
+
+    after do
+      Spree::State.where(name: 'Berat').delete_all
+      Spree::Country.where(name: 'Albania').delete_all
+      Spree::Country.where(name: 'Chad').delete_all
+    end
```

Comment:
It looks like that code is very old from the very early days: 1288760839f1f1c9f10dec8bd3d8fc92c3b68107

I remember that there was some difficulty with letting the cleaner delete these, because of foreign key constraints. We would also need a method to provide countries and states for the tests that need them.

So yes, if you would like to create a pull request to change it, great. This exception is really not nice. It is very annoying when you did a database migration and then reset the test database. At the moment, you need to quit Zeus and start it again to run the country seeding. Otherwise tests just fail.